.github/workflows/publish-to-confluence.yml

```diff
@@ -0,0 +1,101 @@
+# uses https://github.com/justmiles/go-markdown2confluence to publish a README.md or other specified document(s) to Confluence
+name: publish-to-confluence-reusable-workflow
+on:
+  workflow_call:
+    inputs:
+      comment:
+        description: Add an optional comment to the published page.
+        required: false
+        type: string
+      exclude:
+        description: List of exclude file patterns (regex) that will be applied on markdown file paths.
+        required: false
+        type: string
+      modified_since:
+        description: Only upload files that have modifed in the past n minutes.
+        required: false
+        type: number
+      parent:
+        description: Optional parent page to nest content under.  Separate multiple parent pages with /.
+        required: false
+        type: string
+      space:
+        description: Confluence space under which content should be published.  Default TXM.
+        required: false
+        default: TXM
+        type: string
+      title:
+        description: Set the page title on upload (defaults to filename without extension).
+        required: false
+        type: string
+      use_document_title:
+        description: Will use the Markdown document title (# Title) if available.  Default false.
+        required: false
+        default: false
+        type: string
+      documents:
+        description: The documents to upload.  Can be a directory of documents or a single file.  Default is README.md.
+        required: false
+        default: README.md
+        type: string
+
+    outputs:
+      confluence_url:
+        value: ${{ jobs.publish-to-confluence.outputs.confluence_url }}
+
+jobs:
+  publish-to-confluence:
+    runs-on: ubuntu-latest
+
+    container:
+      image: nexus.txm.tools/golang:latest
+      credentials:
+        username: ${{ secrets.NEXUS_USER_CI_DOCKER }}
+        password: ${{ secrets.NEXUS_PASSWORD_CI_DOCKER }}
+      env:
+        CONFLUENCE_USERNAME: opex@tempus-ex.com
+        CONFLUENCE_PASSWORD: ${{ secrets.MARKDOWN_CONFLUENCE }}
+        CONFLUENCE_ENDPOINT: https://epf.atlassian.net/wiki
+
+    permissions: write-all
+
+    steps:
+      - name: download and install markdown2confluence
+        run: |
+          curl -LO https://github.com/justmiles/go-markdown2confluence/releases/download/v3.4.6/go-markdown2confluence_3.4.6_linux_x86_64.tar.gz
```

Comment:
There's a good chance that we want to mirror this and ensure availability.

Reply:
fair point.  Let me look into getting this into nexus.